src/utils/makeFragment.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Create a DocumentFragment and fill it with HTML from a string
+ *
+ * @param {string} htmlString - A string of valid HTML
+ * @returns {DocumentFragment}
+ */
+export default function makeFragment(htmlString) {
+  const tempDiv = document.createElement('div');
+
+  tempDiv.innerHTML = htmlString.trim();
+
+  const fragment = document.createDocumentFragment();
+
+  fragment.appendChild(...Array.from(tempDiv.childNodes));
```

Comment:
appendChild accepts just one node
```suggestion
  fragment.append(...Array.from(tempDiv.childNodes));
```